### PR TITLE
NAS-104153 / 11.2 / Added support for the /dev/twsX (3ware 9750) controller

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/common/smart/smartctl.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
     if driver.startswith("ciss"):
         return [f"/dev/{driver}{controller_id}", "-d", f"cciss,{channel_no}"]
 
-    if driver.startswith("twa"):
+    if any(driver.startswith(s) for s in ("twa", "twe", "tws")):
```
Comment (william-gr) [inline src/middlewared/middlewared/common/smart/smartctl.py]:
```
>>> 'twa'.startswith(("twa", "twe", "tws"))
True
>>> 'twe'.startswith(("twa", "twe", "tws"))
True
>>> 'tws'.startswith(("twa", "twe", "tws"))
True
>>> 'tww'.startswith(("twa", "twe", "tws"))
False

```

Comment (william-gr) [reply src/middlewared/middlewared/common/smart/smartctl.py]: You're gonna have to translate that now :P

Comment (themylogin) [reply src/middlewared/middlewared/common/smart/smartctl.py]: LENIN-To-Learn 1  (1)

**Comment by william-gr** [comment]
backport

**Pull request merged**